tests/functional/tests/nrpe_tests.py

```diff
@@ -43,15 +43,6 @@ def test_01_nrpe_check(self):
         nrpe_checks = {
             "check_conntrack.cfg": "command[check_conntrack]="
             "/usr/local/lib/nagios/plugins/check_conntrack.sh",
-            "check_space_root.cfg": "command[check_space_root]="
```

Comment:
There is no kvm support on the arm64 runners (self-hosted or githhub), so we can't launch lxd VMs.  Because of this, I needed to remove all tests that assume a VM or physical machine.

Reply:
Now, I'm seeing this. Can we make this smart to use VM for amd64 and run this test?

Reply:
I guess we could figure something out, but the directive from @Pjack is to not spend extra time on tests because this charm is deprecated / maintenance mode.